libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/TarResponseErrorMap.kt

```diff
@@ -0,0 +1,32 @@
+package com.mapbox.services.android.navigation.v5.navigation
+
+import java.util.HashMap
+import okhttp3.ResponseBody
+import retrofit2.Response
+
+internal class TarResponseErrorMap(private val errorCodes: HashMap<Int, String>) {
```

Comment:
`TarResponseErrorMap` is only instantiated in `TarFetchedCallback` and it's passing a plain `errorCodes` `HashMap`, should we remove this parameter and create it at `init` time?

Reply:
need for tests, I've added default param `private val errorCodes: HashMap<Int, String> = HashMap()`